roles/redhat_satellite6_capsule_lb/handlers/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
@Smithe2413 if there is nothing in the handlers file then just delete the directory